simple/stats/config.py

```diff
@@ -83,6 +86,17 @@ def import_type(self, input_file_name: str) -> ImportType:
           f"Unsupported import type: {import_type_str} ({input_file_name})")
     return ImportType(import_type_str)
 
+  def format(self, input_file_name: str) -> ImportType | None:
+    format_str = self._input_file(input_file_name).get(_FORMAT_FIELD)
+    if not format_str:
+      return None
+    if format_str not in iter(InputFileFormat):
+      raise ValueError(f"Unsupported format: {format_str} ({input_file_name})")
+    return InputFileFormat(format_str)
+
+  def column_mappings(self, input_file_name: str) -> dict[str, str]:
+    return self._input_file(input_file_name).get(_COLUMN_MAPPINGS_FIELD, {})
```

Comment:
should self._input_file(input_file_name) be computed once and stored as a map in "__init__.py"? This is repeatedly doing a few times in each function.

Reply:
Good observation. It is computed once and cached: https://github.com/datacommonsorg/import/blob/e8b0ada6e9a494698c9d26fd9ebafcba350e3f5b/simple/stats/config.py#L57-L61